### PR TITLE
first setup project configuration

### PR DESCRIPTION
**Pull request by edopore**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```
Comment (manuelq12) [inline .github/CODEOWNERS:1]: End of FIle (EOF)

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (edopore) [reply .github/CODEOWNERS]: I made the correction, thanks a lot

File: .gitignore [hunk 2]
```diff
@@ -5,6 +5,7 @@
 # gitignore template for the CypressIO, browser test framework
 # website: https://www.cypress.io/
 
+cypress/
```
Comment (holgiosalos) [inline .gitignore:8]: the modification should be more specific, you should add two paths:
- `cypress/e2e/1-getting-started/*`
- `cypress/e2e/2-advanced-examples/*`

Comment (edopore) [reply .gitignore]: I added those folders to .gitignore file
Thx for you advise

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**